be/src/util/doris_metrics.cpp

```diff
@@ -161,6 +158,10 @@ DorisMetrics::DorisMetrics() : _name("doris_be"), _hook_name("doris_metrics"), _
     REGISTER_DORIS_METRIC(tablet_cumulative_max_compaction_score);
     REGISTER_DORIS_METRIC(tablet_base_max_compaction_score);
 
+    REGISTER_DORIS_METRIC(cache_memory_total);
```

Comment:
```suggestion
    REGISTER_DORIS_METRIC(query_cache_memory_total_mb);
```

Reply:
query_cache_memory_total is  byte，REGISTER_DORIS_METRIC(query_cache_memory_total);